### PR TITLE
ui notification component

### PR DESCRIPTION
**Pull request by bariscc**
implemented dismissable notification component

### CODE REVIEW

File: src/components/ui/notification.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+import {
+  Slash, AlertCircle, CheckCircle, Info, X,
+} from 'react-feather';
```
Comment (SelmanKahya) [inline src/components/ui/notification.js]: lets import Icon component from ui/icon.js and use it

File: src/components/ui/notification.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        className={cls(style.common, style.container[type], extraClassName)}
+        type={type}
+      >
+        {icons[type]}
+        <div className={cls(style.textArea)}>
+          {title && <Paragraph extraClassName="font-extrabold mb-2 text-lg">{title}</Paragraph>}
+          <Paragraph>{text}</Paragraph>
+          {buttons}
+        </div>
+        <X className="text-lgray cursor-pointer" onClick={() => { this.dismiss(); }}/>
```
Comment (SelmanKahya) [inline src/components/ui/notification.js]: looking at the designs:
https://app.zeplin.io/project/5be16ddeab2d281f31bba667/screen/5beaa2f5d4ba093cf83daac2

there are also non-dismissable notification boxes, can we have a prop? `dismissable: true | false`

File: src/components/ui/notification.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+UINotification.propTypes = {
+  buttons: PropTypes.oneOfType([
+    PropTypes.object,
+    PropTypes.arrayOf(PropTypes.object),
+  ]),
+  extraClassName: PropTypes.string,
+  text: PropTypes.string.isRequired,
+  title: PropTypes.string,
+  type: PropTypes.oneOf(['success', 'warning', 'danger', 'info']).isRequired,
```
Comment (SelmanKahya) [inline src/components/ui/notification.js]: for button component, we call this styleType, lets keep it consistent

**Comment by SelmanKahya** [comment]
looks really clean! can we merge it?

**Comment by bariscc** [comment]
oops, missed that top comment. unit test and screenshots are on the way

**Comment by bariscc** [comment]
notif1
notif2

added test as well

File: test/components/ui/notification.test.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    wrapper.setProps({ styleType: 'info' });
+    expect(wrapper.hasClass('border-blue')).toEqual(true);
+    wrapper.setProps({ styleType: 'warning' });
+    expect(wrapper.hasClass('border-yellow')).toEqual(true);
+  });
+
+  test('is dismissable', () => {
+    wrapper.setProps({ dismissable: true });
+
+    expect(wrapper.find('Icon').filterWhere((item) => {
```
Comment (SelmanKahya) [inline test/components/ui/notification.test.js]: ideally, we can simulate an event (click on X icon) and make sure notification is hidden after click

**Pull request merged**